INSTALLATION.md

```diff
@@ -1,3 +1,30 @@
+# Requirements and limitations
+
+## Target platform
+
+ This application is designed to run on IPhones with IOS. In addition, the application can be used on phones with Android OS.  But the display style of the app interface may differ from the usual, because the elements for creating the interface of this application designed with IOS-style. Also, correct display and use of the app on tablets is not guaranteed and depends on the support of their compatibility with Flutter-apps.
+
+## System requirements and limitations
+
+ For correct operation of applications on the ***IPhone*** you need:
```

Comment:
For correct operation of **the application** on ***IPhone*** **the following is required**: